catalog/core/catalog-core-commands/pom.xml

```diff
@@ -199,22 +199,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.39</minimum>
+                                            <minimum>0.30</minimum>
```

Comment:
@spearskw was this just to get builds to pass during development? ValidateCommand actually gets a 98% code coverage from Jacoco with the current tests
![screen shot 2016-06-27 at 9 35 49 am](https://cloud.githubusercontent.com/assets/12501737/16387713/c7c32676-3c4a-11e6-8ae9-f4db4b0fa097.png)